core/kogitoservice/dashboards.go

```diff
@@ -24,10 +24,10 @@ import (
 
 	"github.com/kiegroup/kogito-operator/core/infrastructure"
 
-	grafanav1 "github.com/integr8ly/grafana-operator/v3/pkg/apis/integreatly/v1alpha1"
 	"github.com/kiegroup/kogito-operator/api"
 	"github.com/kiegroup/kogito-operator/core/client/kubernetes"
 	"github.com/kiegroup/kogito-operator/core/framework"
+	grafanav1 "github.com/kiegroup/kogito-operator/core/infrastructure/grafana/v1alpha1"
```

Comment:
maybe upgrade this api?

Reply:
`v1alpha1` is the latest grafana-operator api.